### PR TITLE
Fixed behat typos

### PR DESCRIPTION
**Pull request by wachterjohannes**
Main PR: https://github.com/sulu-io/sulu-docs/pull/26

### CODE REVIEW

File: tests/Sulu/Component/Content/Mapper/ContentMapperTest.php [hunk 1]
```diff
@@ -1479,17 +1479,17 @@ public function testDelete()
         $subChild = $this->mapper->save($data[3], 'overview', 'default', 'de', 1, true, null, $child->getUuid());
 
         $this->eventDispatcher->expects($this->at(0))
-            ->method('dispatch')
-            ->with(
-                $this->equalTo(ContentEvents::NODE_PRE_DELETE),
-                $this->isInstanceOf('Sulu\Component\Content\Event\ContentNodeDeleteEvent')
-            );
+        ->method('dispatch')
```
Comment (dantleech) [inline tests/Sulu/Component/Content/Mapper/ContentMapperTest.php]: Bad indentation.

Comment (dantleech) [inline tests/Sulu/Component/Content/Mapper/ContentMapperTest.php]: Also, this isn't related to this PR is it?

**Comment by wachterjohannes** [comment]
@dantleech no there is a mistake (=

**Comment by wachterjohannes** [comment]
@dantleech new PR here https://github.com/sulu-io/sulu/pull/1237

**Pull request closed without merging**